package.json

```diff
@@ -11,7 +11,9 @@
     "format": "prettier --config .prettierrc.json --write 'src/**/*.js'",
     "format:check": "prettier --config config/.prettierrc.json --debug-check --list-different 'src/**/*.js'",
     "lint": "eslint 'src/**/*.js'",
-    "lint:fix": "eslint --fix 'src/**/*.js'"
+    "lint:fix": "eslint --fix 'src/**/*.js'",
+    "parcel": "parcel index.html",
```

Comment:
can we call this yarn script `parcel:dev` to provide a hint about what it does (starts a dev server)?